ndmg/register.py

```diff
@@ -266,6 +267,13 @@ def gen_tissue(self):
         print("Extracting brain from raw T1w image...")
         reg_utils.t1w_skullstrip(self.t1w, self.t1w_brain, self.skull)
 
+        #  QA part of skull strip
+        self.namer.dirs["qa"]["skull_strip"] = f'{self.namer.dirs["qa"]["base"]}/skull_strip'
+        if not os.path.exists(self.namer.dirs["qa"]["skull_strip"]):
+            os.makedirs(self.namer.dirs["qa"]["skull_strip"])
+        print('QA_skullstrip_path  ', self.namer.dirs["qa"]["skull_strip"])
+        gen_overlay_pngs(self.t1w_brain, self.t1w, self.namer.dirs["qa"]["skull_strip"])
```

Comment:
this stuff looks fine. I currently have a [branch](https://github.com/neurodata/ndmg/tree/exorcise-namer) where I'm going to remove namer from the pipeline, but we can remove it here ourselves when it's ready

Reply:
I guess keep namer at this version will deduce mistake...maybe replace namer by one time is better? So that we can keep the coding style and habit continually.